Assets/ScreenSwipe/ScreenSwipe.cs

```diff
@@ -174,7 +178,7 @@ private IEnumerator CheckForOrientationChange()
 				// refresh contents on the change
 				RefreshContents();
 			}
-			yield return null;
+			yield return waitForEndOfFrame;
```

Comment:
I didn't realise there was difference between these. But yes this would be better to use. Thanks I will add this .

Reply:
I was wrong on this, if you do `yield return null` it runs after `Update()`, but `yield return waitForEndOfFrame` runs at the end of the frame. It doesn't run as quick as possible like i thought it did

[Here is what i found on the subject](https://answers.unity.com/questions/755196/yield-return-null-vs-yield-return-waitforendoffram.html)

Reply:
Just read that same thing. I still think end of frame make sense here though.